privacyidea/lib/error.py

```diff
@@ -146,6 +148,17 @@ def __init__(self, description="CA error!", id=ERROR.CA):
         privacyIDEAError.__init__(self, description=description, id=id)
 
 
+class CSRError(privacyIDEAError):
```

Comment:
I think they should inherit from the CAError so we have one base error.

Reply:
done